src/styles/blocks/card.scss

```
@@ -0,0 +1,116 @@
+$color_white: #fff;
+$color_black: #060b35;
+$color_blue: #00acdc;
+$color_grey: #616070;
+$radius: 5px;
+$button_width: 166px;
+$button_height: 40px;
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  font-family: Roboto, Helvetica, sans-serif;
+
+}
+
+.card {
+  width: 200px;
+  height: 406px;
+  flex-shrink: 0;
+  border-radius: $radius;
+  border: 1px solid #f3f3f3;
+  background: $color_white;
+  margin: auto;
+  padding: 32px 16px 16px;
+
+  &__foto {
+    display: block;
+    margin: 0 auto 40px;
+    width: 160px;
+    height: 134px;
+    background-size: cover;
+    background-position: center;
+  }
+
+  &__text {
+    color: $color_black;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    margin: 0 0 4px;
+  }
+
+  &__code {
+    color: $color_grey;
+    font-size: 10px;
+    line-height: 14px;
+    margin: 0 0 16px;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    margin: 24px 0 16px;
+    color: $color_black;
+    text-align: right;
+    font-size: 16px;
+    font-weight: 700;
+    height: 18px;
+    align-items: center;
+  }
+
+  &__price1 {
+    color: $color_grey;
+    font-size: 12px;
+    line-height: 18px;
+    font-weight: 400;
+  }
+
+  &__reviews {
+    display: flex;
+    justify-content: flex-end;
+    width: 53px;
+    height: 16px;
+    flex-direction: column;
+    flex-shrink: 0;
+    color: $color_black;
+    text-align: right;
+    font-size: 10px;
+    line-height: 14px;
+  }
+
+  &__star {
+    display: flex;
+    justify-content: space-between;
+  }
+
+  &_a {
+    display: flex;
+    justify-content: center;
+    text-align: center;
+    align-items: center;
+    text-decoration: none;
+    color: $color_white;
+    font-size: 14px;
+    font-weight: 700;
+    line-height: 16px;
+    text-transform: uppercase;
+    width: $button_width;
```

should be a full parent's width
```suggestion
    width: 100%;
```